src/main/java/nextstep/ladder/domain/ladder/Height.java

```diff
@@ -0,0 +1,22 @@
+package nextstep.ladder.domain.ladder;
+
+public class Height {
+
+    private static final int MINIMUM_VALID_HEIGHT = 1;
+    private int height;
```

Comment:
```suggestion
    private static final int MINIMUM_VALID_HEIGHT = 1;
    
    private int height;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가하면 가독성이 상승합니다!